packages/static/kairos-overlay-files/files/etc/cos/bootargs.cfg

```diff
@@ -1,14 +1,18 @@
 function setSelinux {
     source (loop0)/etc/os-release
-    set baseSelinuxCmd=""
-    if test $KAIROS_FAMILY == "rhel" -o test $ID == "opensuse-tumbleweed" -o test $ID == "opensuse-leap"; then
-        set baseSelinuxCmd="selinux=0"
-    else
-        # if not in recovery
-        if [ -z "$recoverylabel" ];then
-            set baseSelinuxCmd="security=selinux selinux=1"
-        fi
-    fi
+
+    # Disable selinux for all distros. Supporting selinux requires more than
+    # just enabling it like this.
+    set baseSelinuxCmd="selinux=0"
+
+    #if test $KAIROS_FAMILY == "rhel" -o test $ID == "opensuse-tumbleweed" -o test $ID == "opensuse-leap"; then
```

Comment:
since this is applied to all other distros, could it be that it will break any of the boards? since those are not tested, thinking mainly about nvidia 🤷

Reply:
I can't think of a reason why a board would require selinux to work :shrug: